rmf_scheduler_msgs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rmf_scheduler_msgs</name>
+  <version>0.0.0</version>
```

Comment:
Nit, this should probably be changed to match the version of all the other packages in the repo?

Reply:
updated